### DISCUSSION TITLE
Inviting someone to private repo

### DISCUSSION THREAD

**Discussion by tinni95** [community/community#3151] [Mobile] [answered]
I was not able to invite someone to join a private repository through the mobile app

**Comment by rnystrom** [comment, marked as answer]
Thanks for the feedback! There are a _ton_ of features on GitHub.com that the app doesn't support yet. The mobile app serves as a companion to using GitHub.com. For now I would recommend using the website to invite and accept private repository invitations.

**Reply by natrist** [reply]
2 years later I still have to authenticate through a browser even though my app is logged in. Could it be added as an option or is it too unsafe?

**Reply by stan-spotts** [reply]
The response included "for now" four years ago. Has this request made it nearer to consideration for implementation now?

**Comment by foaed1234** [comment]
يجيب علا اي مستخدم أن يحسن أعماله 
ولا يعتقد أن السلوك السيء ذي نتيجه فا احسن عملك ودقنه

**Comment by foaed1234** [comment]
تطوير بنية تحتيه للاساس والقواعد
